style.css

```diff
@@ -642,9 +662,7 @@ option.sr-select-item {
   .period-information {
     display: none;
   }
-  .period.current {
-    left: 0px;
-  }
```

Comment:
not sure why this was removed, I think it looks better if it was left in

Reply:
woah college very cool indeed

Reply:
yeah i think everything in this pr that deals with the ui is kinda goofy
i agree with u